### ISSUE TITLE
[ERROR] add_special_tokens = True not working in version 3.0.0

### ISSUE THREAD

**Issue by 1512262** [huggingface/transformers#5499]
# 🐛 Bug

## Information

Model I am using (Bert, XLNet ...): BERT

Language I am using the model on (English, Chinese ...): Multi-Lingual

The problem arises when using:
* [x ] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x ] my own task or dataset: (give details below)

## To reproduce

In version 2.11 Everything is OK but I updated to 3.0.0 and I can not create fixed_length with padding for the encoding of the text

## Environment info

     
- `transformers` version:
- Platform:
- Python version:
- PyTorch version (GPU?):
- Tensorflow version (GPU?):
- Using GPU in script?:
- Using distributed or parallel set-up in script?:

**Comment by thomwolf** [comment]
Hi, can you provide more information (basically fill all the filed in the issue templates) in particular provide a clear example so we can try to reproduce the behavior?
